### PR TITLE
DTSPB-3783 pp iht transformation

### PR DESCRIPTION
**Pull request by IswaryaPepakayala**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-3783
https://tools.hmcts.net/jira/browse/DTSPB-3784
https://tools.hmcts.net/jira/browse/DTSPB-3785
https://tools.hmcts.net/jira/browse/DTSPB-3786
https://tools.hmcts.net/jira/browse/DTSPB-3847
https://tools.hmcts.net/jira/browse/DTSPB-3859
https://tools.hmcts.net/jira/browse/DTSPB-3820
https://tools.hmcts.net/jira/browse/DTSPB-3826
https://tools.hmcts.net/jira/browse/DTSPB-3828
### Change description ###

HMRC IHT PP and Caseworker changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseField.json [hunk 1]
```diff
@@ -3524,6 +3553,10 @@
   {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "primaryApplicantPhoneNumber", "UserRole": "caseworker-probate-rparobot", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "primaryApplicantEmailAddress", "UserRole": "caseworker-probate-rparobot", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "deceasedNameSection", "UserRole": "caseworker-probate-rparobot", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "valueOfAssetSection", "UserRole": "caseworker-probate-rparobot", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "hmrcLetterId", "UserRole": "caseworker-probate-rparobot", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "uniqueProbateCodeId", "UserRole": "caseworker-probate-rparobot", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "uniqueProbateCodeId", "UserRole": "caseworker-probate-systemupdate", "CRUD": "CRUD"},
```
Comment (DelythJustice) [inline ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseField.json]: this one's popped up in the wrong place

File: ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json [hunk 2]
```diff
@@ -9,11 +9,12 @@
...
   {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "IHT205", "ListElement": "IHT205 - there was no inheritance tax to pay", "DisplayOrder": 1},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "IHT207", "ListElement": "IHT207 - the person who died lived abroad and there was no inheritance tax to pay", "DisplayOrder": 2},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "IHT400421", "ListElement": "IHT400 and IHT421", "DisplayOrder": 3},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "NA", "ListElement": "Not applicable - DO NOT USE", "DisplayOrder": 4},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormEstate", "ListElementCode": "IHT207", "ListElement": "IHT207", "DisplayOrder": 1},
-  {"LiveFrom": "02/01/2017", "ID": "IHTFormEstate", "ListElementCode": "IHT400421", "ListElement": "IHT400 and IHT421", "DisplayOrder": 2},
+  {"LiveFrom": "02/01/2017", "ID": "IHTFormID", "ListElementCode": "IHT400", "ListElement": "IHT400 - only select this option if you applied after ((date))", "DisplayOrder": 2},
```
Comment (DelythJustice) [inline ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json]: think was agreed that date here would be a variable, maybe like CCD_DEF_CASE_SERVICE_BASE_URL

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json [hunk 3]
```diff
@@ -480,5 +480,9 @@
...
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "removedRepresentative", "Label": "Removed Representative", "FieldType": "RemovedRepresentative", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "changeOfRepresentatives", "Label": "Change Of Representatives", "FieldType": "Collection", "FieldTypeParameter": "ChangeOfRepresentative", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "changeOfRepresentative", "Label": "Change Of Representative", "FieldType": "ChangeOfRepresentative", "SecurityClassification": "Public"}
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "changeOfRepresentative", "Label": "Change Of Representative", "FieldType": "ChangeOfRepresentative", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "valueOfAssetSection", "Label": "The values you enter must be whole numbers, and must not include decimals, fullstops and commas.For example 25000, not 25,000.", "FieldType": "Label", "SecurityClassification": "Public"},
```
Comment (DelythJustice) [inline ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json]: this has missed the space after the full stop, should be:
. For example

File: ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseField.json (hunk 1 above)
Comment (IswaryaPepakayala) [reply ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseField.json]: Modified

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json (hunk 3 above)
Comment (IswaryaPepakayala) [reply ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json]: Modified

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json [hunk 4]
```diff
@@ -146,7 +146,7 @@
...
...with will annexed (admon will) online when certain conditions apply.\n\n<a href=\"https://www.gov.uk/guidance/probate-paper-applications-for-legal-professionals\" target=\"_blank\">Guidance on exemptions, conditions and when application must be submitted by paper</a>\n\nStep by step guidance on submitting an application online can be found at <a href=\"https://www.gov.uk/government/publications/myhmcts-how-to-apply-for-probate-online/apply-for-probate-with-myhmcts\" target=\"_blank\">Apply for probate with MyHMCTS</a>\n\n## If you have submitted IHT400 and IHT421 forms before {{ date }}\nYou must wait 20 working days after sending the IHT400 and IHT421 forms to HMRC before you apply for the grant.\n\nIf you do not wait, we will be unable to verify the IHT421 details from HMRC.This will delay the grant being issued.\n\n## If you have submitted an IHT400 form after{{ date }}\nYou must wait to receive a letter from HMRC with a probate code.You will need this code to apply.\n\nHMRC will send out the letter up to 20 working days after you submit the IHT400 form. If you have not received the letter,you should contact HMRC. \n\n", "FieldType": "Label", "SecurityClassification": "Public"},
```
Comment (DelythJustice) [inline ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json]: probably should have space after , in:
,you should contact HMRC
Apologies, didn't notice this first time

File: ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json (hunk 2 above)
Comment (DelythJustice) [reply ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json]: too much effort

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json (hunk 4 above)
Comment (DelythJustice) [reply ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json]: I've put in the space

**Review by DelythJustice — APPROVED** [review]
functional tests failing but code approved

File: Jenkinsfile_CNP [hunk 5]
```diff
@@ -232,7 +232,7 @@ withPipeline("java", product, component) {
 
   afterSuccess('functionalTest:preview') {
     if (checkForEnableNightlyLabel(env.BRANCH_NAME)) {
-      env.TEST_E2E_URL = "https://xui-probate-back-office-pr-${CHANGE_ID}.preview.platform.hmcts.net"
+      env.TEST_E2E_URL = "https://manage-case.demo.platform.hmcts.net"
```
Comment (FeliTam) [inline Jenkinsfile_CNP]: Need to revert this line before merging(and remove nightly label)

Comment (DelythJustice) [reply Jenkinsfile_CNP]: I think only remove nightly label

Comment (DelythJustice) [reply Jenkinsfile_CNP]: If we put in demo to get this to work for e2e

**Pull request merged**